### DISCUSSION TITLE
Enabling SpellCheck in HtmlContentEditor

### DISCUSSION THREAD

**Discussion by DrastiGohel** [serenity-is/Serenity#6734] [Help]
I want to enable spellcheck in HtmlContentEditor.cs 
For that I am creating custom editor class: (this is in Namespace folder)
![image](https://user-images.githubusercontent.com/98172041/225941859-dc58094a-6aae-4df2-a328-a622dcda25d0.png)

But when I try to use the editor in Form.cs/Row.cs it is not working: (this file is in Module)

![image](https://user-images.githubusercontent.com/98172041/225942208-1a05b0a3-6af4-4acf-995e-42743f2d6805.png)

Serenity Version is 6.4

**Comment by LSNicholls** [comment]
Do you have something like this in your Form.cs/Row.cs file?
```
import {CustomHtmlContentEditor } from '../../CustomHtmlContentEditor'; 
//probably this is not correct file reference, but something similar
```
